apps/voting/app/src/hooks/useVotes.js

```diff
@@ -12,7 +12,8 @@ function useDecoratedVotes() {
   const installedApps = useInstalledApps()
 
   return useMemo(() => {
-    if (!votes) {
+    // Make sure we have loaded information about the current app and other installed apps before showing votes
+    if (!(votes && currentApp && installedApps)) {
```

Comment:
Should this be part of this PR?

Reply:
ahh no, i needed that in order to test it, we need to get merged https://github.com/aragon/aragon-apps/pull/1116